script/rsync.sh

```diff
@@ -36,7 +36,7 @@ set +e
     [ -e "$subfolder/print_openqa.sh" ] || { >&2 echo "No file found: {$subfolder/print_openqa.sh}"; exit 1; }
 
     # nowhere to log yet as we haven't created $logfolder
-    bash -e "$subfolder/read_files.sh"
+    bash -e "$subfolder/read_files.sh" ||  { >&2 echo "read_files.sh failed for $ubfolder in enviroment $environ"; exit 1; }
```

Comment:
```suggestion
    bash -e "$subfolder/read_files.sh" ||  { >&2 echo "read_files.sh failed for $subfolder in enviroment $environ"; exit 1; }
```

Where is `$environ` defined? I don't see it in this repo 🤔

Reply:
in line 4 of this script

Reply:
Oh, thanks. Of course `grep $environ` failed me 🤦🏾‍♀️

Reply:
thx, typo :D